tests/autoyast/installation.pm

```diff
@@ -118,6 +118,12 @@ sub run {
             $num_errors++;
         }
         elsif (match_has_tag('warning-pop-up')) {
+            # See poo#19978, no timeout on partition warning, hence need to click OK button to soft-fail
+            if (check_screen('warning-partition-reduced', 0)) {
```

Comment:
It's exclusive, and it's a warning, so will match both tags. In general we can use match_has_tag, but for soft-fail I don't want to add it to expected needles array unless you insist.